Formula/f/flang.rb

```diff
@@ -37,10 +38,13 @@ def install
     relative_resource_dir = resource_dir.relative_path_from(llvm.prefix.realpath)
 
     common_args = %W[
+      -GNinja
       -DBUILD_SHARED_LIBS=ON
       -DLLVM_DIR=#{llvm.opt_lib}/cmake/llvm
       -DLLVM_ENABLE_FATLTO=ON
       -DLLVM_ENABLE_LTO=ON
+      -DLLVM_RAM_PER_COMPILE_JOB=5000
+      -DLLVM_RAM_PER_LINK_JOB=10000
```

Comment:
Helped on arm64 macOS (3 -> 1.5 hour).

14-x86_64 was slower (1 -> 1.75 hour); however, may just be an unlucky run as 13-x86_64 is about the same and LLVM will limit the max jobs to number of logical cores. So 64GB / 5GB ~ 12 jobs should be ignored.

May try a couple more things

Reply:
For comparison, Fedora approximates 6.114 GB/process though that seems to be applied to total processes like our `MAKEFLAGS` -  https://src.fedoraproject.org/rpms/flang/blob/rawhide/f/flang.spec#_72.